test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -50,62 +50,64 @@ int createRandomTest(std::vector<char*> const& _parameters)
 	bool debug = false;
 	bool filltest = false;
 
+	TestOutputHelper::initTest(1);
 	dev::test::Options& options = const_cast<dev::test::Options&>(dev::test::Options::get());
 
 	testSuite = options.rCurrentTestSuite;
-	if (testSuite != "BlockChainTests" && testSuite != "TransactionTests" && testSuite != "StateTests"
+	if (testSuite != "BlockchainTests" && testSuite != "TransactionTests" && testSuite != "StateTestsGeneral"
 				&& testSuite != "VMTests")
-		testSuite = "";
+	{
+		std::cerr << "Error! Test suite '" + testSuite +"' not supported! (Usage -t TestSuite)" << std::endl;
+		return 1;
+	}
 
 	for (size_t i = 0; i < _parameters.size(); ++i)
 	{
 		auto arg = std::string{_parameters.at(i)};
 
-		if (arg == "--fulloutput")
-			options.fulloutput = true;
-		else if (arg == "--debug")
+		if (arg == "--debug")
 			debug = true;
 		else
 		if (arg == "--filldebug")
 			filldebug = true;
 	}
 
-	if (testSuite == "")
+	if (checktest)
+		std::cout << "Testing: " << testSuite.substr(0, testSuite.length() - 1) << "... ";
+
+	if (testSuite == "StateTestsGeneral")
 	{
-		std::cerr << "Error! Test suite not supported! (Usage -t TestSuite)" << std::endl;
-		return 1;
+		if (checktest)
+			return checkRandomTest(dev::test::doStateTests, testmValue, debug);
```

Comment:
this is a legacy code and actually is not working. 
there was an option to run the test provided by std input. 

this PR enables the support of 
```./testeth -t StateTestsGeneral -- --createRandomTest```
command 
other options like --checkstate are yet not supported. but I could review the code when needed.